source/documentation/modules/color.md

```diff
@@ -2,17 +2,18 @@
 title: sass:color
 ---
 
-<%= partial '../snippets/built-in-module-status' %>
-
-<% function <<SIGNATURE, 'adjust-color(...)', returns: 'color' do
-color.adjust($color,
-  $red: null, $green: null, $blue: null,
-  $hue: null, $saturation: null, $lightness: null,
-  $whiteness: null, $blackness: null,
-  $alpha: null)
-SIGNATURE
-%>
-  <% impl_status dart: '1.28.0', libsass: false, ruby: false, feature: "$whiteness and $blackness" %>
+{% render 'doc_snippets/built-in-module-status' %}
+
+{% capture color_adjust %}
+  color.adjust($color,
+    $red: null, $green: null, $blue: null,
+    $hue: null, $saturation: null, $lightness: null,
+    $whiteness: null, $blackness: null,
+    $alpha: null)
+{% endcapture %}
+
+{% function color_adjust, 'adjust-color(...)', 'returns:color' %}
```

Comment:
LiquidJS does not allow inline args to include newlines, but this `{% capture %}` pattern can be used to accomplish the same thing.